### PR TITLE
:bug: Display tags color upon their category

### PR DESCRIPTION
**Pull request by gildub**
PF Label support the following 8 colors "gold" | "red" | "blue" | "cyan" | "green" | "orange" | "purple" | "grey" .
Tag colors depends upon their TagCategory colors which are defined as either keyword names (blue, etc) or hexadecimal values (for the recent ones V3).

Therefore depending on the color of the tag the Label can be displayed with a supported color or will fall back to "grey" (Label default) when not supported.

LabelCustomColor from migtools/lib-ui wraps Label to make tag color supported through CSS properties.

image

Resolves https://issues.redhat.com/browse/MTA-332

### CODE REVIEW

**Review by ibolton336 — CHANGES_REQUESTED** [review]
Need to rethink this one together. Probably will need input from UXD and Ramon here.

**Comment by mturley** [comment]
This might help :) https://github.com/migtools/lib-ui/pull/125

**Comment by gildub** [comment]
@mturley, thanks and nice work with the migtools/lib-ui wrapper.
Good luck with the backend change in that regard.

File: client/src/app/pages/applications/components/application-tags/application-tags.tsx [hunk 1]
```diff
@@ -222,22 +223,19 @@ export const ApplicationTags: React.FC<ApplicationTagsProps> = ({
                       </Text>
                     </TextContent>
                     <Flex>
-                      {tagsInThisCategoryInThisSource
-                        ?.sort((a, b) => a.name.localeCompare(b.name))
```
Comment (mturley) [inline client/src/app/pages/applications/components/application-tags/application-tags.tsx:226]: @gildub why remove the sorting here?

**Review by mturley — APPROVED** [review]
LGTM, left a question we can follow up on later.

**Pull request merged**